charts/openfga/templates/tests/test-connection.yaml

```diff
@@ -11,5 +11,5 @@ spec:
     - name: wget
       image: busybox
       command: ['wget']
-      args: ['{{ include "openfga.fullname" . }}:{{ .Values.service.port }}']
+      args: ['--spider', '{{ include "openfga.fullname" . }}:{{ .Values.service.port }}/healthz']
```

Comment:
`--spider` makes it so wget doesn't download anything, and point to `/healthz` to return something other than a 404

Reply:
Given that the HTTP server is not guaranteed to be running (e.g. you can disable it), I think it would be better if we used the grpc health probe. Also, the HTTP server requires the gRPC server to be running anyways, so the common denominator is the gRPC health probe. This is also more consistent with what we do elsewhere in our ecosystem. For example,
https://github.com/openfga/helm-charts/blob/main/charts/openfga/templates/deployment.yaml#L292

Could you please change this to use the following instead?
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: "{{ include "openfga.fullname" . }}-test-connection"
  labels:
    {{- include "openfga.labels" . | nindent 4 }}
  annotations:
    "helm.sh/hook": test
spec:
  containers:
    - name: grpc-health-probe
      image: ghcr.io/grpc-ecosystem/grpc-health-probe:v0.4.20
      args: ['-addr={{ include "openfga.fullname" . }}:{{ (split ":" .Values.grpc.addr)._1 }}']
  restartPolicy: Never

```